aod-discord-bot.js

```diff
@@ -319,6 +322,20 @@ function messageReply(message, msg, edit) {
 }
 global.messageReply = messageReply;
 
+function sendGlobalNotification(message, msg) {
```

Comment:
Sorry to yank you around. Can we just pass guild directly here instead of message? I don't see a reason we need message at all.

Reply:
fixed in 95acb07